src/main/resources/assets/idwtialsimmoedm/lang/ru_ru.json

```
@@ -0,0 +1,47 @@
+  "enchantment.minecraft.fire_aspect.desc": "Поджог противника.",
+  "enchantment.minecraft.looting.desc": "Повышение вероятности выпадения редкой добычи и её количества.",
+  "enchantment.minecraft.efficiency.desc": "Повышение скорости ломания.",
+  "enchantment.minecraft.silk_touch.desc": "Выпадение стекла, руд и других блоков в неизменном виде при разрушении.",
+  "enchantment.minecraft.unbreaking.desc": "Уменьшение вероятности потери прочности при использовании предмета.",
+  "enchantment.minecraft.fortune.desc": "Повышение вероятного количества выпадающих при разрушении предметов из руд, культур и других блоков.",
+  "enchantment.minecraft.power.desc": "Увеличение урона от стрел при попадании из лука.",
+  "enchantment.minecraft.punch.desc": "Усиление откидывания противника при попадании из лука.",
+  "enchantment.minecraft.flame.desc": "Поджог стрелы и противника при попадании в него.",
+  "enchantment.minecraft.infinity.desc": "Бесконечная стрельба из лука обычными стрелами при наличии в инвентаре хотя бы одной.",
+  "enchantment.minecraft.luck_of_the_sea.desc": "Увеличение вероятности получения сокровищ при рыбалке.",
+  "enchantment.minecraft.lure.desc": "Снижение длительности попадания рыбы на крючок.",
+  "enchantment.minecraft.depth_strider.desc": "Увеличение скорости перемещения в воде.",
+  "enchantment.minecraft.frost_walker.desc": "Замораживает воду под игроком.",
+  "enchantment.minecraft.mending.desc": "Использует опыт для починки какого-либо предмета в руках или в слотах брони.",
+  "enchantment.minecraft.binding_curse.desc": "Данное зачарование на предмете не позволит вам выбросить или снять его со слота брони, выпадает из игрока только в случае его смерти.",
+  "enchantment.minecraft.vanishing_curse.desc": "Предмет с данным зачарованием самоуничтожится при смерти игрока.",
+  "enchantment.minecraft.sweeping.desc": "Увеличивает урон по мобам, стоящих рядом с целью.",
+  "enchantment.minecraft.loyalty.desc": "Позволяет трезубцу автоматически возвращаться после броска.",
+  "enchantment.minecraft.impaling.desc": "Увеличивает трезубцу урон по мобам, которые естественным образом появляются в океане.",
+  "enchantment.minecraft.riptide.desc": "Позволяет трезубцу запускать игрока вперед. Работает только во время дождя или когда вы находитесь в воде.",
+  "enchantment.minecraft.channeling.desc": "Позволяет трезубцу вызывать молнии во время шторма.",
+  "enchantment.minecraft.multishot.desc": "При выстреле из арбалета одновременно вылетают 3 стрелы в четко определенных направлениях.",
+  "enchantment.minecraft.quick_charge.desc": "Увеличивает скорость перезарядки арбалета.",
+  "enchantment.minecraft.piercing.desc": "Позволяет стрелам пробивать несколько существ насквозь.",
+  "enchantment.minecraft.soul_speed.desc": "Увеличивает скорость передвижения на песке душ и почве душ.",
+  "enchantment.minecraft.swift_sneak.desc": "Увеличивает скорость передвижения при подкрадывании."
+}
```

Добавил полное и краткое описание в ModMenu.
```suggestion
,  "modmenu.summaryTranslation.idwtialsimmoedm": "Я не хотел устанавливать библиотеку, поэтому сделал свой мод для описаний чар.",
  "modmenu.descriptionTranslation.idwtialsimmoedm": "Лекговесный и независимый мод на описания в любых подсказках, содержащих чары. Используется тот же формат, что и у Enchantment Descriptions, поэтому моды с описаниями своих чар также будут поддерживаться."
}

```